### PR TITLE
[vizbuilder] update to latest canon-core and blueprint

### PR DESCRIPTION
**Pull request by jhmullen**
This branch updates the vizbuilder package to Blueprint 3.  I updated all the packages and paths, changed the appropriate props for new Components, and @frabarz did excellent work in updating his custom select components to play nicely with the new Popover/Select components.

My most recent commit pulls in the `master` branch so this PR could be made.

@frabarz - As the vizbuilder expert, Could you please pull the latest changes to this branch (now that I have pulled in master) to be sure that everything is working as you expect?   If so, I think that this PR can be reviewed and merged (not sure if that final merge will be done by @davelandry or @frabarz) 

Thanks!

### CODE REVIEW

File: packages/vizbuilder/package.json [hunk 1]
```diff
@@ -17,25 +17,28 @@
...
     "fast-sort": "^1.5.3",
     "lodash": "^4.17.10",
+    "memoize-one": "^5.0.4",
     "mondrian-rest-client": "^1.1.0",
-    "pluralize": "^7.0.0",
+    "pluralize": "^8.0.0",
     "prop-types": "^15.6.0",
     "query-string": "^6.1.0",
-    "react-tiny-virtual-list": "^2.1.4",
+    "react": "^16.8.6",
```
Comment (davelandry) [inline packages/vizbuilder/package.json]: this shouldn't be added as a direct dependency? (it should be whichever version comes from canon)

**Comment by davelandry** [comment]
@frabarz @jhmullen this wiill probably also need another pull from `master`, bringing in the bug fixes in #572 that I just merged in

**Comment by frabarz** [comment]
I'm going to fix a bug on the main measure selector, after that it should be ready.

File: packages/vizbuilder/package.json (hunk 1 above)
Comment (davelandry) [reply packages/vizbuilder/package.json]: @frabarz can you speak to this? (especially after the convos on Slack)

Comment (frabarz) [reply packages/vizbuilder/package.json]: fixed on b43d576540dd00d5549870f6b8d8d104e28c25b9

**Pull request merged**